eng/service-templates/ProductConstructionService/provision.bicep

```
@@ -1,18 +1,17 @@
 @minLength(1)
 @description('Primary location for all resources')
-param location string = 'northcentralus'
-param stageLocation string = 'northcentralusstage'
+param location string = 'westus2'
 
 @minLength(5)
 @maxLength(50)
 @description('Name of the Azure Container Registry resource into which container images will be published')
 param containerRegistryName string = 'productconstructionint'
 
 @description('CPU cores allocated to a single container instance, e.g., 0.5')
-param containerCpuCoreCount string = '0.25'
+param containerCpuCoreCount string = '0.5'
 
 @description('Memory allocated to a single container instance, e.g., 1Gi')
-param containerMemory string = '0.5Gi'
+param containerMemory string = '1Gi'
```

I think we should do at least 2Gi. Git can take quite some memory when working with large repositories.
LibGit2Sharp takes even more.

```suggestion
param containerMemory string = '2Gi'
```

---

done, we can increase it even further. We'll also have to increase the CPU core count tho https://learn.microsoft.com/en-us/azure/container-apps/containers#:~:text=Consumption%20workload%20profile-,0.25,2.5Gi,-%E2%9C%94

---

Yeah, I think 2 and 8 might be feasible eventually but let's revisit later. For now, I am pretty sure we'd hit 1Gi, 2Gi should be okay